### ISSUE TITLE
Screen Reader Table Navigation Broken

### ISSUE THREAD

**Issue by aardrian** [duetds/date-picker#27]
**Describe the bug**
The calendar grid identifies as a table/grid to my screen reader, but drops me into a mode where arrow keys choose between dates instead of letting me move word-by-word or character-by-character. If I use table navigation (`Ctrl` + `Alt` + arrow keys in JAWS), every cell reports as the end of the row or column.

If I back out of the grid and press `T` to access it as a table, I can at least navigate the header row (the days of the week) with table navigation, but I cannot get into the grid of dates.

**To Reproduce**
1. Go to the grid,
2. Use `Ctrl` + `Alt` + arrow keys to move within the grid,
3. Observe it says "end of row" or "bottom of column" or similar;
4. Press `Shift` + `Tab` to back out of the grid,
5. Press `T` to move to the table,
6. Use `Ctrl` + `Alt` + down arrow to move into days,
3. Observe it says "bottom of column".

**Expected behavior**
I should be able to navigate the entire calendar grid using my screen reader's table navigation controls.

**Screenshots**
I made a video, but GitHub does not allow me to embed it.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 85
 - JAWS 2020

**Additional context**
Using first example at https://duetds.github.io/date-picker/

**Comment by viljamis** [comment]
@aardrian Thank you for reporting this. We’re aware of this issue, but not yet sure what would be the best way to solve this while keeping the keyboard functionality for non screen reader users as well.

I think we’ll need to do more testing with Jaws here and make sure we won’t break the default functionality. 

Btw, also just added an accessibility audit to the repository which was conducted earlier. That highlights the same issue, but these details you gave will help us quite a lot to dig deeper.

**Comment by WickyNilliams** [comment]
Fixed in #51, released in `v1.2.0`
